README.md

```diff
@@ -1,6 +1,6 @@
-| CicleCI Build Status | Sonar Code Quality | DockerHub Release | DockerHub Release Candidate | DockerHub Develop |
+| GitHub Actions Build Status | Sonar Code Quality | DockerHub Release | DockerHub Release Candidate | DockerHub Develop |
```

Comment:
Let's add another column to the beginning of this table titled "DockerHub Image Build" and use the badge for the [dockerhub.yml](https://github.com/usdot-fhwa-stol/carma-platform/actions/workflows/dockerhub.yml) workflow